src/Pretzel.Tests/Templating/Context/SiteContextGeneratorTests.cs

```diff
@@ -1199,5 +1199,24 @@ public void are_beforeprocessingtransforms_called()
             // assert
             Assert.Equal(2, pageTransformMock.PostCount);
         }
+
+        [Fact]
+        public void supports_datafiles()
+        {
+            fileSystem.AddFile(@"C:\TestSite\_data\people.yml", new MockFileData(@"dave:
+    name: David Smith
+    twitter: DavidSilvaSmith"));
+
+            // act
+            var context = generator.BuildContext(@"C:\TestSite", @"C:\TestSite\_site", false);
+
+            // assert
+            Assert.NotNull(context.Data);
+            Assert.IsType<Data>(context.Data);
+            Assert.NotNull(context.Data["people"]);
+            dynamic data = context.Data;
+            Assert.Equal("David Smith", data["people"]["dave"]["name"]);
+            Assert.Equal("DavidSilvaSmith", data["people"]["dave"]["twitter"]);
+        }
     }
```

Comment:
Is it possible to add a test with data used in a post or page?

Reply:
I can write an integration test, sure.

Reply:
done. I'm not sure about razor, but there the data context will be available via Page.Site.Data[""] anyway or?

Reply:
I believe so. Tests in liquid only are good for now.